core/kernel/classes/class.Resource.php

```diff
@@ -61,17 +61,6 @@ class core_kernel_classes_Resource extends core_kernel_classes_Container
      */
     public $uriResource = '';
 
-    /**
-     * The resource label
-     * direct access to the label is deprecated,
-     * please use getLabel()
-     *
-     * @access public
-     * @var string
-     * @deprecated
```

Comment:
Even though the property usage has been deprecated, that modification is a breaking change.

Reply:
has the performance impact been considered by removing the label cache?